frontend/src/hooks/useBoardOverview.ts

```diff
@@ -0,0 +1,34 @@
+import useUser from './useUser';
+import { useCallback, useEffect, useState } from 'react';
```

Comment:
Was the `supabase-query` library not sufficient for this?

Reply:
It would help us reduce the use of any

Reply:
got rid of the any, and nah I couldn't find a way to skipcond using the library, so I could not find a way to run dependent queries with the library (fetch of boards _depends_ on us having fetched the user id)